api/event.js

```diff
@@ -0,0 +1,88 @@
+const fetch = require('node-fetch')
+const gqlr = require('graphql-request')
+const { GraphQLClient } = gqlr
+const { API_URL } = require('../src/config')
+
+const EndPoint = `${API_URL}/graphql`
+const GetParty = `
```

Comment:
Capitalised snake case for queries e.g. `GET_PARTY`

Reply:
The `GetParty` is copied from `checkWithdraw.js` https://github.com/wearekickback/app/blob/8a5aa9f1962bbc72b7625c170a3dd38fecd42203/scripts/checkWithdraw.js#L12  

Please confirm the recommended name convention, and we can switch to that.

Reply:
Yes please. I believe all my queries in /src/queries are capitalised.